e2e-tests/upgrade-consistency/run

```diff
@@ -110,9 +113,11 @@ main() {
 
     # test 1.6.0
     API="pxc.percona.com/v1-6-0"
+    kubectl_bin apply -f https://raw.githubusercontent.com/percona/percona-xtradb-cluster-operator/release-1.6.0/deploy/rbac.yaml
```

Comment:
@nonemax  why do we need to have it? We apply it while creating infrastructure (deploying operator and so on).

Reply:
because we have https://github.com/percona/percona-xtradb-cluster-operator/blob/v1.5.0/deploy/rbac.yaml#L102 in 1.5.0 and dont have it in 1.6.0

Reply:
From 1.5.0 we started removing release branches after release, so only tag will be available thus this will maybe stop working - but I guess this can be then fixed after release?